docs/guides/auth/index.rst

```diff
@@ -0,0 +1,308 @@
+.. _ref_guide_auth:
+
+====
+Auth
+====
+
+.. toctree::
+    :hidden:
+    :maxdepth: 3
+
+    built_in_ui
+    email_password
+    oauth
+
+:edb-alt-title: Using EdgeDB Auth
+
+EdgeDB Auth is a batteries-included authentication solution for your app built
+into the EdgeDB server. Here's how you can integrate it with your app.
+
+
+Enable extension in your schema
+===============================
+
+Auth is an EdgeDB extension. To enable it, you will need to add the extension
+to your app’s schema:
+
+.. code-block:: sdl
+
+    using extension auth;
+
+
+Extension configuration
+=======================
+
+The best and easiest way to configure the extension for your database is
+to use the built-in UI. To access it, run ``edgedb ui``. If you have the
+extension enabled in your schema as shown above, and if you have migrated that
+schema change, you will see the "Auth Admin" icon in the left-hand toolbar.
+
+.. image:: images/ui-auth.png
+    :alt: The EdgeDB local development server UI highlighting the auth admin
+          icon in the left-hand toolbar. The icon is two nested shield
+          outlines, the inner being a light pink color and the outer being
+          a light blue when selected.
+    :width: 100%
+
+The auth admin UI exposes these values:
+
+
+auth_signing_key
+----------------
+
+The extension uses JSON Web Tokens (JWTs) internally for many operations.
+``auth_signing_key`` is the value that is used as a symmetric key for signing
+the JWTs. At the moment, the JWTs are not considered “public” API, so there is
+no need to save this value for your own application use. It is exposed mainly
+to allow rotation.
+
+
+token_time_to_live
+------------------
+
+This value controls the expiration time on the authentication token’s
+JSON Web Token. This is effectively the “session” time.
+
+
+allowed_redirect_urls
```

Comment:
@raddevon 

Since this parameter is not updatable via the `edgedb ui` (yet i suppose), I tried to update it manually and had hard time doing so.

Finally was able to do it via the configure command:

```
configure current database set ext::auth::AuthConfig::allowed_redirect_urls := {"http://localhost:4000/", "http://localhost:3000/"}
```

Maybe it would be good to add code snippets on how to set each of the property via REPL? 

First I was trying to update the 'table' via regular query, with no luck
```
 update  ext::auth::AuthConfig set { allowed_redirect_urls := "http://localhost:3000/" };
edgedb error: InternalServerError: cannot delete from view "eb39dffd-d950-53ad-9974-c14c22ebd918"
  Server traceback:
      edb.errors.InternalServerError: cannot delete from view "eb39dffd-d950-53ad-9974-c14c22ebd918"
```

So I imagine multiple users could struggle with this as well.